deps/build.jl

```diff
@@ -79,10 +79,9 @@ init_deps()
 end
 
 if is_apple()
-    if Pkg.installed("Homebrew") === nothing
-        error("Homebrew package not installed, please run Pkg.add(\"Homebrew\")")
-    end
     using Homebrew
+    Homebrew.update()
+    Homebrew.add("imagemagick")
```

Comment:
turns out my /usr/local/lib/libimagemagick was from a manual install on my part of imagemagick using homebrew.  BinDeps founds it, but `MAGICK_CONFIGURE_PATH` and `MAGICK_CODER_MODULE_PATH` were set incorrectly by `build.jl` because it assumed it was installed with HomeBrew.jl.

instead of adding a validate function, i [propose](https://github.com/JuliaIO/ImageMagick.jl/pull/56) to add logic to handle the manually installed case.